server/pom.xml

```diff
@@ -454,6 +454,11 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Not needed. If we use Iterables.size instead

Reply:
In next push